src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1778,6 +1787,24 @@ private void checkInternetPermission() {
             throw new IllegalStateException(new AuthenticationException(ADALError.DEVELOPER_INTERNET_PERMISSION_MISSING));
         }
     }
+    
+    private void logBrokerVersion () {
+        final String currentActiveBrokerPackageName = mBrokerProxy.getCurrentActiveBrokerPackageName();
+        if (!StringExtensions.IsNullOrBlank(currentActiveBrokerPackageName)) {
+            final StringBuilder brokerRelatedInfoLogging = new StringBuilder("The active broker is: " 
+                        + currentActiveBrokerPackageName);
+            if (mBrokerProxy.isBrokerWithPRTSupport(currentActiveBrokerPackageName)) {
+                brokerRelatedInfoLogging.append(" The active broker version is: " 
+                        + AuthenticationConstants.Broker.BROKER_PROTOCOL_VERSION 
+                        + ". It contains PRT support.");
+            } else {
+                brokerRelatedInfoLogging.append(" The active broker version is: v1. It doesn't contain "
+                        + "PRT support");
+            }
+            
+            Logger.v(TAG, brokerRelatedInfoLogging.toString());
```

Comment:
I think I get it. StringBuilder is for concatenating large numbers of strings, i don't actually need it in my case. Will update it, Thanks.